micrometer-core/build.gradle

```diff
@@ -36,6 +36,11 @@ dependencies {
 
     compile 'com.squareup.okhttp3:okhttp:3.9.0',optional
 
+    //tracing
+    compile 'io.opentracing:opentracing-api:0.30.0', optional
```

Comment:
re: @adriancole earlier comment that 0.31.0 has a breaking API change.

Reply:
I just upgrade to `0.31.0-RC1` and it does break the API I was using. Should I program against the RC? It appears the `Scope` API is the way forward (and matches the OpenCensus examples I looked at)

Reply:
I really don't know. Let's hang tight and wait for Adrian's input.

Reply:
This PR (https://github.com/opentracing/opentracing-java/issues/189) covers the changes. Looks like they will include a backwards compatible shim. So coding against `0.30.0` is probably the right decision. But I agree that @adriancole 's input will be good.